CompetitionMARS/Tests/CertificationTests.cs

```diff
@@ -0,0 +1,364 @@
+﻿
+using AventStack.ExtentReports.Reporter;
+using AventStack.ExtentReports;
+using CompetitionMARS.Models;
+using CompetitionMARS.Pages;
+using CompetitionMARS.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OpenQA.Selenium.Support.UI;
+using OpenQA.Selenium;
+using SeleniumExtras.WaitHelpers;
+using NUnit.Framework;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+
+namespace CompetitionMARS.Tests
+{
+    public class CertificationTests : CommonDriver
+    {
+        private Certification certPage;
+        public static ExtentReports extent;
+        public List<string> addedCertificationsEntries = new List<string>();
+
+
+        [OneTimeSetUp]
+        public void SetUp()
```

Comment:
your hooks should be moved into a hooks class, for example OneTimeSetUp and OneTimeTearDown should be implemented in hooks your other hooks like setup and teardown can be per test class can you please change this